clubs/README.md

```diff
@@ -174,35 +190,194 @@ keep building their website to make it unique. At the second meeting, you'll
 pick up where you left off and work up to demos, where everyone from the club
 will come to the front and demo what they built at the end of the meeting.
 
-A small but easy way to dramatically improve your first meeting is to bring
-food. Pastries or cookies from a local supermarket are cheap and work well.
-
-Also, to set the tone and culture of your club, you should introduce the
-[code of conduct][coc] in the introduction presentation and emphasize why it's
-important.
-
-[coc]: ../CONDUCT.md
-
-We recommend the following format for your first meeting:
-
-1. Wait for hackers (club members) to arrive (5 minutes)
-2. Introduction presentation (5 minutes)
-3. Learning names activity (10 minutes)
-4. Build your first website workshop (45 minutes)
-5. Free-form hacking (25 minutes)
-6. Reminder with details for second meeting
+We recommend the below format for your first meeting. For each item in the
+schedule, there is a corresponding (and linked) section below. We've also
+created a slideshow presentation to help you run your first meeting.
+[Click here][presentation] to go to it. We'll be referring to this presentation
+throughout the sections below.
+
+1. [Wait for hackers (club members) to arrive (5 minutes)][1]
+2. [Introduction presentation (5 minutes)][2]
+3. [Name game (10 minutes)][3]
+4. [Build your first website workshop (45 minutes)][4]
+5. [Free-form hacking (25 minutes)][5]
+6. [Second meeting reminder][6]
+
+[presentation]: https://docs.google.com/presentation/d/1QhErlJS6rnqu78guPBWuJ1SUyPJKLbxzLNuSN9qVw0c/edit
+[1]: #waiting-for-hackers-to-arrive
+[2]: #introduction-presentation
+[3]: #name-game
+[4]: #first-website-workshop
+[5]: #free-form-hacking
+[6]: #second-meeting-reminder
+
+#### Waiting for Hackers to Arrive
+
+As hackers start to arrive, your goal is to make them feel welcome. We recommend
+doing the following four things to accomplish this:
+
+1. Put the first slide of the previously mentioned presentation on the projector
+2. Put some music on (not too loud, [click here][playlist] for a good pre-made
+   Spotify playlist)
+3. Make people feel welcome as they walk in
+  - Introduce yourself
+  - Ask a couple questions to initiative conversation
+    - "How did you hear about Hack Club?"
+    - "What sorts of things are you interested in learning?"
+  - You want people to feel good about their decision to come to Hack Club
+4. Tell people what will be happening next (ex. "We're going to get the first
+   meeting started in about 10 minutes after people are done trickling in. In
+   the meantime, please make yourself at home and let me know if I can be
+   helpful with anything.")
```

Comment:
Want to make a pull request to add this after this pull request is merged?

On Mon, Jan 18, 2016 at 12:06 PM, Jonathan Leung <notifications@github.com>
wrote:

> In clubs/README.md
> <https://github.com/hackclub/hackclub/pull/657#discussion_r50039110>:
>
> > +As hackers start to arrive, your goal is to make them feel welcome. We recommend
> > +doing the following four things to accomplish this:
> > +
> > +1. Put the first slide of the previously mentioned presentation on the projector
> > +2. Put some music on (not too loud, [click here][playlist] for a good pre-made
> > +   Spotify playlist)
> > +3. Make people feel welcome as they walk in
> > +  - Introduce yourself
> > +  - Ask a couple questions to initiative conversation
> > +    - "How did you hear about Hack Club?"
> > +    - "What sorts of things are you interested in learning?"
> > +  - You want people to feel good about their decision to come to Hack Club
> > +4. Tell people what will be happening next (ex. "We're going to get the first
> > +   meeting started in about 10 minutes after people are done trickling in. In
> > +   the meantime, please make yourself at home and let me know if I can be
> > +   helpful with anything.")
>
> Can you also add a note about facilitating conversation between hackers
> and optimize for facilitation conversations to people who are not already
> talking. So like if someone is sitting by themselves, go and make
> conversation with them and try to bring them into any existing
> conversations.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/657/files#r50039110>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>